app/frontend/src/cljs/teet/project/project_view.cljs

```diff
@@ -221,14 +221,33 @@
 
 (defn add-user-form
   [e! user project-id]
-  [:div
-   [form/form {:e! e!
-               :value user
-               :on-change-event #(e! (project-controller/->UpdateProjectPermissionForm %))
-               :save-event #(e! (project-controller/->SaveProjectPermission project-id user))
-               :spec :project/add-permission-form}
-    ^{:attribute :project/participant}
-    [select/select-user {:e! e! :attribute :project/participant}]]])
+  (let [roles (-> @authorization-check/roles
+                  (disj :admin)
```

Comment:
Is this enforced in the backend as well?

Reply:
I meant to put this pr down as WIP, the authorization matrix needs permissions on who can give out what permission.

Reply:
Roger that.

Reply:
For now, added assertion on backend side that admin can't be given.